xprof/btx_timeline_params.yaml

```diff
@@ -1,2 +1,4 @@
 :type: map
-:entries: []
+:entries:
```

Comment:
I do not think this feature should be introduced on this PR, since we are doing nothing with this param at all.

Reply:
I doesn't work, we need a new version of metababel (see some PR on metababel)

Reply:
I will take a look on that :)